kge/cli.py

```diff
@@ -272,9 +279,10 @@ def main():
             dataset = Dataset.load(config)
 
             # let's go
-            job = Job.create(config, dataset)
             if args.command == "resume":
-                job.resume(checkpoint_file)
+                job = Job.resume(checkpoint_file, config=config)
```

Comment:
dataset not used here (generally, dataset argument should be optional)

Reply:
fixed